closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -367,6 +373,54 @@ private static void checkFIPSCompatibility(Properties props) {
         }
     }
 
+    /**
+     * Check whether security property can be set.
+     *
+     * A security property that is set by a RestrictedSecurity profile,
+     * while FIPS security mode is enabled, cannot be reset programmatically.
+     *
+     * Every time an attempt to set a security property is made, a check is
+     * performed. If the above scenario holds true, a SecurityException is
+     * thrown.
+     *
+     * One can override this behaviour and allow the user to set any security
+     * property through the use of `-Dsemeru.fips.allowsetproperties=true'.
```

Comment:
Suggest using `{@code ...}` or discuss setting the system property.

Reply:
Used `{@code ...}`.